aperture.go

```diff
@@ -209,6 +209,14 @@ func getConfig(configFile string) (*config, error) {
 	if cfg.ListenAddr == "" {
 		return nil, fmt.Errorf("missing listen address for server")
 	}
+
+	for _, service := range cfg.Services {
+		if service.Price < 0 {
+			return nil, fmt.Errorf("negative price set for "+
+				"service %s", service.Name)
+		}
```

Comment:
Could you also add a check for the max? `lnd` prevents creating invoices above 100,000 BTC.

Reply:
Is it fine if i use 'lnwire.MaxMilliSatoshi.ToSatoshis() as a max?

Reply:
It can still fail since `lnd` enforces the 100,000 BTC maximum at the RPC level, which is less than `lnwire.MaxMilliSatoshi.ToSatoshis()`.